examples/Makefile.am

```diff
@@ -1,27 +1,29 @@
-noinst_PROGRAMS = meshlinkapp chat chatpp manynodes channels monitor groupchat
+noinst_PROGRAMS = meshlinkapp chat manynodes channels groupchat
 
 AM_CPPFLAGS = $(PTHREAD_CFLAGS) -I${top_srcdir}/src -iquote. -Wall
 AM_LDFLAGS = $(PTHREAD_LIBS)
 
 meshlinkapp_SOURCES = meshlinkapp.c
-meshlinkapp_LDADD = ${top_builddir}/src/libmeshlink.la
+meshlinkapp_LDADD = ${top_builddir}/src/libmeshlink.la ${top_builddir}/catta/src/libcatta.la
 
 chat_SOURCES = chat.c
-chat_LDADD = ${top_builddir}/src/libmeshlink.la
+chat_LDADD = ${top_builddir}/src/libmeshlink.la ${top_builddir}/catta/src/libcatta.la
 
-chatpp_SOURCES = chatpp.cc
-chatpp_LDADD = ${top_builddir}/src/libmeshlink.la
+#noinst_PROGRAMS += chatpp
+#chatpp_SOURCES = chatpp.cc
+#chatpp_LDADD = ${top_builddir}/src/libmeshlink.la
 
 manynodes_SOURCES = manynodes.c
-manynodes_LDADD = ${top_builddir}/src/libmeshlink.la
+manynodes_LDADD = ${top_builddir}/src/libmeshlink.la ${top_builddir}/catta/src/libcatta.la
 
 channels_SOURCES = channels.c
-channels_LDADD = ${top_builddir}/src/libmeshlink.la
+channels_LDADD = ${top_builddir}/src/libmeshlink.la ${top_builddir}/catta/src/libcatta.la
 
-if CURSES
+if MONITOR
+noinst_PROGRAMS += monitor
 monitor_SOURCES = monitor.c
 monitor_LDADD = ${top_builddir}/src/libmeshlink.la $(NCURSES_LIBS) -lm
```

Comment:
This also needs to link with `libcatta.la`, just like `chatpp`.

Reply:
Ahh, the issue of `arm-linux-androideabi-g++` unable to find "new.h" header file is still there and I tried to resolve it but didn't find a solution for it, also I tried building on few other ubuntu machines but observed the same problem.
So when I use `#include <new.h>` in `src/meshlink++.h` from where it actually tries to find those header files? Is that predefined in `make-standalone-toolchain.sh` script or should it be defined in autotools also? 
While running that toolchain script according to `README.android` instructions it logs that it's "Copying libstdc++ headers and libraries.." but it seems like the script is not including those header files directory for that toolchain properly. Probably I think `/tmp/my-android-toolchain/arm-linux-androideabi/include/c++/4.6` should be directory from which C++ files pick headers, even that don't have `new.h` and `new.h`'s dependent headers.

You said that for you meshlink compiled successfully, did you use any other commands for building meshlink apart from what's there in `README.android` that I might have missed to do so or do you think any other dependencies there for this toolchain?